test/embind/embind_test.cpp

```diff
@@ -74,6 +75,10 @@ val emval_test_passthrough(val v) {
 void emval_test_return_void() {
 }
 
+std::nullptr_t emval_test_return_nullptr_t() {
```

Comment:
I also don't see this function bound anywhere. e.g. look at how `emval_test_not` works.